src/components/Navbar/types.ts

```diff
@@ -1,10 +1,11 @@
 export interface LinkInfo {
-  href: string
+  href?: string
```

Comment:
Why optional? 🤔

Reply:
The new interface is recursive to account for the dropdown.

```export interface LinkInfo {
  href?: string
  text: string
  dropdown?: LinkInfo[]
}```

At the top level, the menu button does not have an href. Does that make sense?